crossing-repair.lic

```diff
@@ -12,7 +12,7 @@ class CrossingRepair
   def initialize
     setup
 
-    ensure_copper_on_hand(@repair_withdrawal_amount, @bank_info)
```

Comment:
How do you feel about having the ensure_copper_on_hand call in the setup section for now, using the string settings.hometown? The reason I ask is because in setup hometown is data from the get_data call, unlike how I've been using hometown (as the string).

After some of these major changes are implemented I'd like to work on consistency among the scripts in their structure and lookup calls.